ruby/ruby/Ice/ProxyFunctions.rb

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) ZeroC, Inc.
+
+require_relative 'Proxy.rb'
+require_relative 'OperationMode.rb'
+require_relative 'BuiltinSequences.rb'
+
+module ::Ice
+    module ObjectPrx_mixin
```

Comment:
We define the builtin operations here instead of in Proxy.rb to avoid circular dependencies.